4 nivel básico g1

Santiago-2002 opened the pull request:
Entrega NIvel_01

Ya esta funcional el primer nivel con todo lo acordado

Santiago-2002 committed:
Cambios

-Se creo nivel 1
-Se creo primera parte Nivel 1(Plataformas)

mpescobar7301 committed:
cambios nivel 1

se realizaron cambios en al segunda parte del nivel 1

Santiago-2002 committed:
Cambios

-Se realizaron los cambios finales al NIvel 1
-Quedo Finalizado y funcional en todo

Santiago-2002 committed:
Cambios

-Se hicieron cambios en el nivel 1
-Se agregaron materiales y fondos

Santiago-2002 committed:
Update README.md

Se actualizaron los datos del Readme grupo 1

wilmar-arcila reviewed (changes requested):
Cambiar el nombre de la escena de referencia de forma que no haya conflicto con otras que ya tiene el mismo nombre "newSceneReference 1.asset" -> "SceneReference L1G1.asset"

Santiago-2002 committed:
Cambios Grupo1

-Se realizaron cambios en los nombres de la escena Nivel_01
-También en el SceneReference, se modificaron los nombres correctamente

Santiago-2002 committed:
Cambios

-Se agregaron assets de terreno y arboles

wilmar-arcila committed:
Merge branch 'develop' into 4-nivel-básico-g1

wilmar-arcila commented:
Aprobado e incorporado en la rama **develop**
